Corona-Warn-App/src/main/res/menu/menu_statistics_local_incidence.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:id="@+id/menu_information"
```

Comment:
Do these pop up in `R.id.menu` among all other IDs? Would it be useful here to make them easier to find?

Reply:
these id's have been used in other overflow menus before, i though this would keep it consistent 🤔